src/components/logger/request-log.ts

```diff
@@ -103,23 +105,28 @@ export class RequestLog {
 
     probeMsg = `${this.iteration} id:${this.probe.id} ${
       this.response?.status || '-'
-    } ${this.request.method} ${this.request.url} ${
+    } ${this.request?.method} ${this.request?.url} ${
       this.response?.responseTime || '-'
     }ms`
+
     // generate logs from ping request or probe request
-    probeMsg = this.request?.ping
-      ? `${this.iteration} id:${this.probe.id} PING:${
-          this.response?.alive ? 'alive' : 'dead'
-        } ${this.request?.url} ${
-          this.response?.alive ? this.response?.responseTime : '-'
-        }ms packetLoss:${
-          this.response?.alive ? this.response?.packetLoss : '-'
-        }%`
-      : `${this.iteration} id:${this.probe.id} ${
-          this.response?.status || '-'
-        } ${this.request?.method} ${this.request?.url} ${
-          this.response?.responseTime || '-'
-        }ms`
+    if (this.request?.ping) {
+      probeMsg = `${this.iteration} id:${this.probe.id} PING:${
+        this.response?.alive ? 'alive' : 'dead'
+      } ${this.request?.url} ${
+        this.response?.alive ? this.response?.responseTime : '-'
+      }ms packetLoss:${this.response?.alive ? this.response?.packetLoss : '-'}%`
+    } else {
+      probeMsg = flags.verbose
+        ? `${probeMsg} 
+      Request Headers:
+      ${JSON.stringify(this.request?.headers) || '-'} 
+      Request Body:
```

Comment:
Suggestion:  Put hese in one line, more compact and easier to read imho
```javascript
Request Headers: $(JSON.stringify(this.request?headers) || '-'}
```

Then the key and value are in one line:
![image](https://user-images.githubusercontent.com/964106/187014010-b4dee351-c4f3-4aaf-b7ab-c4e7d7342b50.png)

vs

![image](https://user-images.githubusercontent.com/964106/187013952-9c5b80b5-ac87-4e1a-b922-ea926889ede4.png)